README.md

```diff
@@ -156,19 +156,54 @@ Please see [Getting Started](./docs/getting-started.md) for an introduction to t
 
 ## Configuration
 
-Please see the documentation below for details about the ORT configuration.
-
-* The [ORT configuration](./model/src/main/resources/reference.conf) file - the main configuration file for the
-  operation of ORT. This configuration is maintained by an administrator who manages the ORT instance. In contrast to
-  the configuration files in the following, this file rarely changes once ORT is operational.
-* The [.ort.yml](./docs/config-file-ort-yml.md) file - project-specific license finding curations, exclusions
-  and resolutions to address issues found within a project's code repository.
-* The [package configuration](./docs/config-file-package-configuration-yml.md) file - package (dependency) and provenance
-  specific license finding curations and exclusions to address issues found within a scan result for a package.
-* The [curations.yml](./docs/config-file-curations-yml.md) file - curations correct invalid or missing package metadata
-  and set the concluded license for packages.
-* The [resolutions.yml](./docs/config-file-resolutions-yml.md) file - resolutions allow *resolving* any issues
-  or policy rule violations by providing a reason why they are acceptable and can be ignored.
+The following provides an overview over the various configuration files that can be used to customize ORT behavior.
+
+[ORT configuration file](./model/src/main/resources/reference.conf)
+
+The main configuration file for the operation of ORT. This configuration is maintained by an administrator who manages
+the ORT instance. In contrast to the configuration files in the following, this file rarely changes once ORT is
+operational.
+
+| Format | Scope | Default location | Default value |
+| ------ | ----- | ---------------- | ------------- |
+| HOCON | Global | `$ORT_CONFIG_DIR/ort.conf` | Empty ([built-in](./model/src/main/resources/default.conf)) |
+
+[Curations file](./docs/config-file-curations-yml.md)
+
+A file to correct invalid or missing package metadata, and to set the concluded license for packages.
+
+| Format | Scope | Default location | Default value |
+| ------ | ----- | ---------------- | ------------- |
+| YAML / JSON | Global | `$ORT_CONFIG_DIR/curations.yml` | Empty (n/a) |
+
+[Resolution file](./docs/config-file-resolutions-yml.md)
+
+Configurations to resolve any issues or rule violations by providing a mandatory reason, and an optional comment to
+justify the resolution.
+
+| Format | Scope | Default location | Default value |
+| ------ | ----- | ---------------- | ------------- |
+| YAML / JSON | Global | `$ORT_CONFIG_DIR/resolutions.yml` | Empty (n/a) |
+
+[Repository configuration file](./docs/config-file-ort-yml.md)
+
+A configuration file, usually stored in the project's repository, for license finding curations, exclusions and issue
+resolutions.
```

Comment:
"issue and rule violation resolutions"

Reply:
Do you mean we should be more detailed here WRT the *type* of resolution supported? Then we'd need to do the same for the types of excludes, or? And I'd prefer not to go into such detail here, that's what the linked docs are for, IMO.

Reply:
Ah I guess you mentioned this because there's similar wording for `Resolution file`. I'll come up with something.

Reply:
My point was that it sounds like only issues can be resolved if they are eplicitly mentioned but rule violations are not. I think you could just as well also just write "resolutions".